### ISSUE TITLE
Figure out if we need to bring in more settings from Kodi

### ISSUE THREAD

**Issue by keithah** [plexinc/plex-for-kodi#6]
Need to discuss adding more settings options inside of the settings so the user doesn't have to switch back and forth between Kodi and the addon.

**Comment by andyblac** [comment]
it does this already ?, just setup in normal Kodi videos option before launching Plex Add-on.

**Comment by keithah** [comment]
@andyblac what I mean is, should we put that setting in the Plex Settings so you don't have to go into Kodi to configure it.

**Comment by danjames92** [comment]
Absolutely. I don't think anyone doesn't like choice. Refresh rate switching, Audio passthrough options (like in PMP), skip delay would be welcome additions.
